### ISSUE TITLE
Cannot display WebP image

### ISSUE THREAD

**Issue by ToLengSon** [rs/SDWebImage#2429]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 11.0 / 10.13.0 / 11.0 / 4.0
 SDWebImage Version      | 4.4.2
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9 / Xcode 8
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I use SDWebImage v4.4.2, but the bug still exist.

**Comment by dreampiggy** [comment]
Please provide the information in the default template, or we could not help with it. See [ISSUE_TEMPLATE](https://github.com/rs/SDWebImage/blob/master/.github/ISSUE_TEMPLATE.md)

At least, you must speicfy SD version, Platform, OS version and the step to reproduce this issue.

**Comment by bpoplauschi** [comment]
@ToLengSon thanks for reaching out regarding an issue. Now it's important for us to understand your context that you fill in the template info @dreampiggy mentioned above. I added it back to the issue description, so please fill in those details.
For your particular issue, can you reproduce this with our demo? Do you have a url of a webp image you can share that is good to reproduce this issue?

**Comment by ToLengSon** [comment]
Sorry, i'm mistake, my company's server is bad. now, i will close the issue.
